test/hack/resource/pkg/resourcetypes/oidc.go

```diff
@@ -110,3 +106,17 @@ func (o *OIDC) Cleanup(ctx context.Context, arns []string) ([]string, error) {
 	}
 	return deleted, errs
 }
+
+// Retrieve all the OpenID Connect (OIDC) providers in the current AWS account.
+// The AWS IAM service has a default soft limit of 100 OIDC providers per account.
+// The ListOpenIDConnectProviders API call used in this function automatically handles pagination,
+// so the function can retrieve more than 100 OIDC providers if needed, without the caller
+// having to worry about pagination.
+func (o *OIDC) getAllOIDCs(ctx context.Context) (oidcs []iamtypes.OpenIDConnectProviderListEntry, err error) {
```

Comment:
The more I think about it. I don't think this is necessary

Reply:
Ok, I'll leave oidc.go as it was.